### ISSUE TITLE
Installing Kibana before installing Sense unclear on this page https://www.elastic.co/guide/en/elasticsearch/guide/current/running-elasticsearch.html#sense

### ISSUE THREAD

**Issue by tjaensch** [elastic/elasticsearch-definitive-guide#490]
Before it's possible to install Sense like described here https://www.elastic.co/guide/en/elasticsearch/guide/current/running-elasticsearch.html#sense Kibana needs to be installed which is not inherently obvious if you don't already know that. It took me awhile to figure this out before I could proceed with installing Sense...

**Comment by Ij888** [comment]
I agree with @tjaensch. Following [this video](https://info.elastic.co/2016-03-AB-Test-Getting-Started-ES_Video.html?aliId=41942463), it does not say Kibana needs to be installed. 

I'm on a Windows10 machine, running ES v2.3.3. Marvel-agent, License and Head is installed.
When I try to run localhost:9200/_plugins/marvel on the browser, nothing but some line saying this shows up.

So this brings me to the question: 
1. Is Sense included in both Marvel and Kibana?
2. Do I need both Marvel and Kibana to operate Elasticsearch?

**Comment by chiptus** [comment]
I would remove the all installing sense part, and add a part about intalling kibana

**Comment by salvisb** [comment]
+1 to the original post.
